library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -472,21 +481,38 @@ public int removeDownloads(long... ids) {
      * it was running, and it will no longer be accessible through the download manager.
      * If there are any downloaded files, partial or complete, they will be deleted.
      *
-     * @param ids the IDs of the batches to remove
+     * @param batchesIds the IDs of the batches to remove
      * @return the number of batches actually removed
      */
-    public int removeBatches(long... ids) {
-        if (ids == null || ids.length == 0) {
-            throw new IllegalArgumentException("called with nothing to remove. input param 'ids' can't be null");
+    public int removeBatches(long... batchesIds) {
```

Comment:
So the shall rename the method as well? `removeBatchIds`? It looks like a batch can have several ids instead of that there are batches with ids... wasn't sure about that